.github/workflows/test.yaml

```diff
@@ -29,6 +29,11 @@ jobs:
       - name: Copy empty .env.example to .env
         run: |
           cp .env.example .env
+      - name: Run linting
+        run: |
+          black --check .
+          flake8
+          isort . --check
       - name: Run the unit tests
         run: |
           python manage.py test
```

Comment:
Pourquoi, pour tous les autres fichiers, y a une newline à la fin mais on en ajoute pas pour les tests de workflow ? Je vois qu'il y en a pas dans les autres repos sur lesquels j'ai bossé non plus. 🤔

Reply:
Compte-rendu de notre conversation après recherches des deux côtés : 
- la règle UNIX est qu'une ligne se termine par un caractère newline `\n`.
- donc on aime bien ajouter un caractère `\n` à la fin de la dernière ligne pour qu'elle soit complète.
- donc j'ai ajouté un `\n` à la fin du fichier yaml ci-dessus.

[Lien vers la discussion stackoverflow sur la règle](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) (Merci Marie)

Comment configurer vscode pour qu'il ajoute un caractère `\n` à la fin de tous les fichiers : chercher "final newline" dans les paramètres et cocher "insert final newline".
![Capture d’écran 2023-02-08 à 10 38 03](https://user-images.githubusercontent.com/1035145/217494538-3eda6ead-06f0-4a7d-b451-d94e488c7223.png)